engine/src/main/resources/templates/AnsibleServiceOnAws/notifyCustomerProvisionSuccessEmailBodyV2.html

```diff
@@ -0,0 +1,13 @@
+{@boolean renderSection1=true}
+{@boolean renderButtonSection1=true}
+{#include Common/insightsEmailBody}
+{#content-title}
+    Ansible Automation Platform Service on AWS - Ansible Automation Platform
+{/content-title}
+{#content-title-section1}
+    Notify customer environment is ready
```

Comment:
Wouldn't this be something like "Environment is ready"?

Reply:
You are right, that's better, even for a draft :)